rts/at/util/src/main/java/org/jboss/jbossts/star/logging/atI18NLogger.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2021, Red Hat, Inc.
+ *
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * This is free software; you can redistribute it and/or modify it
```

Comment:
@ochaloup do you know why this extra text was added?

Reply:
Yes, the text was added as I was not clear how form the header should have. We established it this way. If that was wrong please create a PR when you'll be adding the information into README. Sorry for the inconveniences if there is some.